server/src/userManagement.mjs

```diff
@@ -228,11 +228,21 @@ function addUser(userId) {
     ws.on('pong', async hb => {
       heartbeat(ws)
     });
-    associateUserWithWs(''+userId, ws);
-    handleGroupMembership(''+userId)
-    ws.on('message', async message => {
-      messageHandler.handleMessage(message, ''+userId, ws);
-    });
+    if ((config.multiUserConnections == 'allow') || (config.multiUserConnections == 'warn') || (config.multiUserConnections == 'deny' && getWsForUser(userId) == undefined)) {
```

Comment:
We should clean up this logic a bit. It looks like it will give the warning log even on the first connection?

Maybe we can put the denial logic at the beginning to get that out of the way. Let's also close() the incoming ws connection so the client knows we're not accepting the connection.

After that we can give a warning log if needed but either way let the connection through.